services/frontend/src/components/Header.js

```diff
@@ -0,0 +1,18 @@
+import PropTypes from "prop-types";
+
+const Header = ({ city, state }) => {
+  return (
+    <div className="md:text-4xl pb-5">
+      <h1>
+        {city}, {state}
+      </h1>
+    </div>
+  );
+};
+
+Header.propTypes = {
```

Comment:
React components accept arguments in the form of a `props` object. But because it's JS it's hard to specify what attributes `props` should/can contain and what types they are. This code block is one way to enforce a level of parameter validation on React.

Basically we're saying that `props` should contain two properties (city and state), they're both string types, and they're both required.

This adds type hints to the editor and also will throw a runtime warning when you render it. It's not as secure as something like TypeScript, but it's a lot faster.